Client/Application/AppDelegate.swift

```diff
@@ -11,6 +11,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var browserViewController: BrowserViewController!
     weak var profile: BrowserProfile?
     var tabManager: TabManager!
+    let keyboardManager = BluetoothKeyboardManager()
```

Comment:
I basically want a shared instance of BluetoothKeyboardManager in the app, which will have a weak reference to all its delegates. There are 2 ways of doing that that I know:
1- Create and store the instance in the appDelegate and retrieve it from UIApplication.sharedApplication().delegate, which is what I'm doing.
2- Use a singleton